plugins/skip-button-plugin/src/skip-button.js

```diff
@@ -0,0 +1,74 @@
+import pillarbox from '@srgssr/pillarbox-web';
+
+/**
+ * @ignore
+ * @type {typeof import('video.js/dist/types/button').default}
+ */
+const Button = pillarbox.getComponent('Button');
+
+/**
+ * Represents a SkipButton component for the pillarbox player.
+ *
+ * @class SkipButton
+ */
+class SkipButton extends Button {
+  /**
+   * @type {TextTrackCue}
+   */
+  activeInterval;
+
+  constructor(player, options) {
+    super(player, options);
+    this.addClass('pbw-skip-button', 'vjs-hidden', 'vjs-visible-text');
+    this.player().on('srgssr/interval', ({ data }) => this.handleTimeIntervalChange(data));
```

Comment:
I would suggest something like:
```javascript
this.handleTimeIntervalChange = this.handleTimeIntervalChange.bind(this);
//Or something like
this.handleTimeIntervalChange_ = ({ data }) => this.handleTimeIntervalChange(data);
```
The reason is if the component is disposed it will produce an error when an interval event is fired:
```javascript
player.getChild('skipButton') .dispose();
```